lib/dmc/base_configuration.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module DMC
```

Comment:
What's `DMC` ? I'd prefer to see a full-on name used for the module here to avoid confusion and inflection issues. We're not short on bytes!